test/parallel/test-cluster-message.js

```diff
@@ -111,7 +111,7 @@ if (cluster.isWorker) {
   // When a TCP server is listening in the worker connect to it
   worker.on('listening', function(address) {
 
-    client = net.connect(address.port, function() {
+    client = net.connect(address.port, '127.0.0.1', function() {
```

Comment:
> Otherwise it will listen on `::1` on an IPv6-enabled system, but will try to connect to `127.0.0.1` and will fail with `ECONNREFUSED`.

Hum that's not what I see on my machine… It tries to connect to `::1` as expected.
If you don't specify a host in `net.connect`, it will use `localhost`, which always resolves to `127.0.0.1` with `verbatim` set to `false` (that's the current test). With `verbatim` set to `true`, it will resolve to `::1` on IPv6 enabled system, and to `127.0.0.1` otherwise.

Anyway, I don't feel strongly about this, if you think it's simpler let's keep IPv4 for this test.

Reply:
> Until we can be sure that every system will also have `::1 localhost` I think the safe solution is to hardcode the address here.

One could argue that the safe thing to do is to keep `verbatim` to `false`… If it's not easy to fix our CI machines, I don't think we can expect our users to fix their machines when updating Node.js.

Reply:
I'm sure if we add `::1 localhost` to `/etc/hosts/` will solve this. Unfortunately I don't have access to SmartOS to examine this. As mentioned already, IPv6 is here to stay and should be accounted for. Having systems with faulty IPv6 will fail eventually. This is a bit ideologic, though, I have to admit. But landing this in the next LTS would give people both the time and the incentive to fix some fundamental issues.